### PR TITLE
[FIX][16.0][event_sale_session]Fix bug #353

### PR DESCRIPTION
**Pull request by percevaq**
Issue #353 

Fix integration issue with sales reports.

### CODE REVIEW

**Comment by percevaq** [comment]
pre-commit I cannot pass it locally with ubuntu 22.04 and python 3.10. There is a known bug with this.
I try to pass it as clean as possible and then the pre-commit above fixes the spaces, lines, etc.

**Comment by pedrobaeza** [comment]
> pre-commit I cannot pass it locally with ubuntu 22.04 and python 3.10. There is a known bug with this.

I have the same stuff and I don't have any problem.

File: event_sale_session/README.rst [hunk 1]
```diff
@@ -77,6 +77,10 @@ Contributors
 
     * Iván Todorovich <ivan.todorovich@gmail.com>
 
+* `GutierrezTi Team <https://gutierrezti.es>`_
+    * Joaquín Gutiérrez Pedrosa <joaquin@gutierrezti.es>
```
Comment (ivantodorovich) [inline event_sale_session/README.rst]: This file is auto-generated from the ones in the `readme` directory.
Changes done here will be overwritten.

Instead, update the `readme/CONTRIBUTORS.rst` file

File: event_sale_session/reports/sale_report.py [hunk 2]
```diff
@@ -27,13 +27,12 @@ class SaleReport(models.Model):
...
-            ", l.event_id as event_id, l.event_ticket_id as event_ticket_id, "
-            "l.event_session_id as event_session_id"
-        )
-        return select
+    def _select_additional_fields(self):
+        res = super()._select_additional_fields()
+        res['event_id'] = 'l.event_id'
+        res['event_ticket_id'] = 'l.event_ticket_id'
+        res['event_session_id'] = 'l.event_session_id'
+        return res
```
Comment (ivantodorovich) [inline event_sale_session/reports/sale_report.py:35]: Technically it looks good, but we do need to get pre-commit to pass.

Could you apply the changes suggested by it, in the ci:

```diff
diff --git a/event_sale_session/reports/sale_report.py b/event_sale_session/reports/sale_report.py
index e7868e4..6a6f937 100644
--- a/event_sale_session/reports/sale_report.py
+++ b/event_sale_session/reports/sale_report.py
@@ -29,9 +29,9 @@ class SaleReport(models.Model):
 
     def _select_additional_fields(self):
         res = super()._select_additional_fields()
-        res['event_id'] = 'l.event_id'
-        res['event_ticket_id'] = 'l.event_ticket_id'
-        res['event_session_id'] = 'l.event_session_id'
+        res["event_id"] = "l.event_id"
+        res["event_ticket_id"] = "l.event_ticket_id"
+        res["event_session_id"] = "l.event_session_id"
         return res
 
     def _group_by_sale(self):
```

**Comment by percevaq** [comment]
> > pre-commit I cannot pass it locally with ubuntu 22.04 and python 3.10. There is a known bug with this.
> 
> I have the same stuff and I don't have any problem.

Hello pedro:

Same in the rest of repositories it works for me but in this one I try to install pre-commit in an env with python 3.10 and I get this error.

`pre-commit run --all-files
[INFO] Installing environment for https://github.com/oca/maintainer-tools.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('python', '-mpip', 'install', '.')
return code: 1
stdout:
    Executable `python` not found
stderr: (none)
`
I don't see a way to fix it.
I have seen that it is something known but I don't know how to get it installed. I have not found a way

File: event_sale_session/reports/sale_report.py (hunk 2 above)
Comment (percevaq) [reply event_sale_session/reports/sale_report.py:35]: Hello:

I am going to cancel the MR, do it again all manual and nothing and see what happens.

Thanks for your comments

Comment (pedrobaeza) [reply event_sale_session/reports/sale_report.py:35]: Let me check if the template is not the latest one in this repository.

**Comment by percevaq** [comment]
Hello:

I think as I have commented the most coherent thing would be to cancel this MR.
Fix everything manually including the suggested improvements and be able to continue with more stuff.

**Review by pedrobaeza — APPROVED** [review]
I have pushed to your branch the proper changes with pre-commit and the README modified in the proper place.

/ocabot merge patch

**Pull request merged**